split.Console/Interfaces/IPlayerInterface.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by an 
+    /// object that will encapsulate all player-related behavior. 
+    /// </summary>
+    public interface IPlayerInterface : ICardInterface
```

Comment:
Would they? IHaveHand seems like it just returns a hand. Does a player do anything else with their hand?